en/docs/integrate/develop/creating-artifacts/creating-a-message-processor.md

```diff
@@ -64,4 +64,4 @@ See the links given below for descriptions of properties for each processor type
 
 ## Tutorials
 
--	See the tutorial on [using message stores and processors]({{base_path}}/integrate/tutorials/storing-and-forwarding-messages)
+-	See the tutorial on [using message stores and processors]({{base_path}}//tutorials/integration-tutorials/storing-and-forwarding-messages)
```

Comment:
Additional slash is there

Reply:
fixed with [33b1850](https://github.com/wso2/docs-apim/pull/7321/commits/33b1850d15ef4f967024552b40b35169ff6bc33c)